workers/unity/Packages/com.improbable.gdk.core/Worker/Worker.cs

```diff
@@ -138,8 +138,6 @@ private void AddCoreSystems()
             World.GetOrCreateManager<WorldCommandsCleanSystem>();
             World.GetOrCreateManager<WorldCommandsSendSystem>();
             World.GetOrCreateManager<CommandRequestTrackerSystem>();
-
-            // MonoBehaviour API
```

Comment:
Why remove this? I don't know when it was changed to say API (as that seems wrong), but it was meant to denote the stuff which isn't actually core but needs to be added here for now.

Reply:
in that case we should already provide a helper class for that as we do for the other modules. will add that

Reply:
actually, just saw that we already have it and we should have removed this line here completely